files/mkafka.c

```diff
@@ -0,0 +1,293 @@
+/* File mailbox.c */
+/* Define system calls to the mailbox */
+
+#include "mkafkalib.h"
+#include <minix/syslib.h>
+#include "glo.h"
+
+
+static mkafka_t *mkafka;
+static int mutex;
```

Comment:
i just realized your "mutex" is an int....

Reply:
:)